package.json

```diff
@@ -61,6 +61,7 @@
     "rimraf": "^2.6.3",
     "rollup": "^2.26.5",
     "rollup-plugin-node-polyfills": "^0.2.1",
+    "rollup-plugin-pnp-resolve": "^2.0.0",
```

Comment:
This isn't necessary as `@rollup/plugin-node-resolve` already works with PnP since it uses `resolve` which we patch. You might however need to set `enableGlobalCache: false`

Reply:
If `GlobalCache` is disabled, the `build` job can no longer reuse the cache prepared in previous steps.

Reply:
Actually it will still populate the global cache. When Yarn installs a package its fetched from the network to the offline mirror(global cache) then if `enableGlobalCache` is false it's copied to the local cache folder (`.yarn/cache`). When using the `node-modules` linker and not checking-in the cache that is an unnecessary step that can be skipped, but for PnP its sometimes required because some resolutions loose the context of which PnP API it belongs to so we need to crawl the filesystem to locate it.

Whether or not the global cache is populated is actually controlled by `enableMirror` https://yarnpkg.com/configuration/yarnrc#enableMirror

Reply:
Rollup will warn that `semver` and many other libraries can not be resolved and thus treated as external libraries: https://github.com/JLHwung/babel/runs/1038268440?check_suite_focus=true#step:6:568

It may relates to a custom rollup plugin that maps `packages/babel-core/lib` to `packages/babel-core/src` so Rollup can do tree shaking: https://github.com/JLHwung/babel/blob/main/scripts/rollup-plugin-babel-source.js

Though test is passed, but I guess the final build does not contain `semver` and the test is passed because we happen to have `semver` in test environments.

Reply:
I see that rollup warning with current `main` branch, so probably unrelated

Reply:
The issue is here https://github.com/babel/babel/blob/8d59ff658defe907ce80816207038bee0f75b2d4/Gulpfile.js#L143-L145 
this causes `@rollup/plugin-node-resolve` to resolve `semver` and `source-map` from the project root which declare neither

Reply:
Thanks! The intention here is to avoid multiple copies of `semver` / `source-map` bundled in `standalone`. I can open new PR to bump these deps so we don't need `dedupe` at all.

Reply:
> If it doesn't work without it the fix is to set enableGlobalCache: false (or the env version for the CI only)

Has been fixed in the latest release so `enableGlobalCache: true` should be fine with PnP now